examples/Constraints_Continuous/interpoint.py

```diff
@@ -0,0 +1,119 @@
+## Example for linear interpoint constraints in a continuous searchspace
+
+# Example for optimizing a synthetic test functions in a continuous space with linear
+# interpoint constraints.
+# While intrapoint constraints impose conditions on each individual point of a batch,
+# interpoint constraints do so **across** the points of the batch. That is, an
+# interpoint constraint of the form ``x_1 + x_2 <= 1`` enforces that the sum of all
+# ``x_1`` values plus the sum of all ``x_2`` values in the batch must not exceed 1.
+
```

Comment:
feels like this misses a sentence giving a concrete example where this could be useful in the real world.

Reply:
Added a sentence, please resolve ifg happy.